emscripten/.gitignore

```diff
@@ -1,3 +1,5 @@
 *.bc
 firebird.js
 *.mem
+*.img
```

Comment:
I'm not sure why I added that .js file to the .gitignore. Probably some kind of fail.